RecoBTag/SecondaryVertex/plugins/BoostedDoubleSVProducer.cc

```diff
@@ -655,8 +655,13 @@ void BoostedDoubleSVProducer::calcNsubjettiness(const reco::JetBaseRef& jet,
             edm::LogWarning("MissingJetConstituent")
                 << "Jet constituent required for N-subjettiness computation is missing!";
         }
-      } else
+      } else {
+        // Check if any values were nan or inf
+        float valcheck = daughter->px() + daughter->py() + daughter->pz() + daughter->energy();
+        if (std::isnan(valcheck) || std::isinf(valcheck))
```

Comment:
As from the warning message displayed by the static analyzer:
"std::isnan / std::isinf does not work when fast-math is used. Please use edm::isNotFinite from 'FWCore/Utilities/interface/isFinite.h'"